src/ListView/ListView.stories.js

```diff
@@ -0,0 +1,62 @@
+import React from 'react'
+import { storiesOf, action } from '@kadira/storybook'
+import { withKnobs, boolean } from '@kadira/storybook-addon-knobs'
+import { defaultTemplate } from '../../storybook/decorators/storyTemplates'
+import MockListManager from './__mocks__/MockListManager'
+import ListView from './ListView'
+import ListViewItem from './ListViewItem'
+import ListViewItemContainer from './ListViewItemContainer'
+import ListViewItemExpansion from './ListViewItemExpansion'
+
+const stories = storiesOf('ListView', module)
+stories.addDecorator(withKnobs)
+stories.addDecorator(
+  defaultTemplate({
+    title: 'List View',
+    documentationLink: 'http://www.patternfly.org/pattern-library/content-views/list-view/'
+  })
+)
+
+stories.addWithInfo(
+  'basic list with expansion',
+  `This is the List View with basic expansion.`,
+  () => <MockListManager />
+)
+
+stories.addWithInfo(
+  'basic list with expansion knobs',
+  `This is the List View with basic expansion.`,
+  () => {
+    const firstLevelExpanded = boolean('First Level Expanded', false)
+    const secondLevelExpanded = boolean('Second Level Expanded', false)
+    return (
+      <ListView>
+        <ListViewItem isActive={firstLevelExpanded}>
+          <ListViewItemContainer
+            isExpansionItem
+            itemClicked={action('first level itemClicked')}
+            isActive={firstLevelExpanded}
+          >
+            First Level
+          </ListViewItemContainer>
+          <ListViewItemExpansion isActive={firstLevelExpanded}>
```

Comment:
Since patternfly depends on bootstrap would it be desirable here to depend on react-bootstrap's [panel accordion](https://react-bootstrap.github.io/components.html#panels-accordion) and utilize that in this component?